test_efs/dsgrid_project/datasets/sector_models/comstock/dimensions.toml

```diff
@@ -15,6 +16,7 @@ file = "dimensions/model_years.csv"
 module = "dsgrid.dimension.standard"
 name = "Model Year 2012"
 type = "model_year"
+display_name = "2012"
```

Comment:
I see not always--trivial dimensions might also list their actual record values ...